src/features/apiDelegation/offered/ChooseApiPage/ChooseApiPage.tsx

```diff
@@ -187,29 +184,16 @@ export const ChooseApiPage = () => {
       >
         <PageHeader icon={<ApiIcon />}>{t('api_delegation.give_access_to_new_api')}</PageHeader>
         <PageContent>
-          {chosenDelegableOrgs.length < 1 ? (
-            <RestartPrompter
-              spacingBottom
-              restartPath={
-                '/' + ApiDelegationPath.OfferedApiDelegations + '/' + ApiDelegationPath.ChooseOrg
-              }
-              title={t('common.an_error_has_occured')}
-              ingress={t('api_delegation.delegations_not_registered')}
-            />
-          ) : (
-            <div>
-              <h3>{t('api_delegation.chosen_orgs')}:</h3>
-              <BorderedList borderStyle={'dashed'}>{chosenDelegableOrgs}</BorderedList>
-            </div>
-          )}
           <h3 className={classes.chooseApiSecondHeader}>
             {t('api_delegation.new_api_content_text2')}
           </h3>
           {isSm && chosenApis.length > 0 && (
```

Comment:
The Skeletons for delegerbare api are shown on smaller screens but not valgte api skeletons and I think that is the correct approach as showing skeletons for both would take up so much space on the page.

![image](https://github.com/Altinn/altinn-access-management-frontend/assets/31471142/f2226a17-2934-4fc8-b409-5b8cbe1840e0)

Reply:
But the thing is, there _are_ skeletons added for the chosen api's, added just a couple of lines down.
Which means that once the first response of the delegationCheck comes back, these skeletons will appear and be on screen until the rest of the responses comes back.
So then we will have some skeletons appear, then some other skeletons will appear, and then all of them will disappear, which will look strange.

Now, considering the other discussion about whether to display skeletons for the chosen APIs at all, this might not be an issue. In that case, we will simply have to make sure nothing is displayed in this section until everything is displayed.